.github/workflows/trivy-analysis.yml

```diff
@@ -22,15 +22,14 @@ jobs:
         uses: aquasecurity/trivy-action@d710430a6722f083d3b36b8339ff66b32f22ee55
         with:
           image-ref: quay.io/keycloak/${{ matrix.container}}:nightly
-          format: template
-          template: '@/contrib/sarif.tpl'
+          format: sarif
           output: trivy-results.sarif
           severity: MEDIUM,CRITICAL,HIGH
           ignore-unfixed: true
-          security-checks: vuln
```

Comment:
@yyvess Out of curiosity, why those security checks were removed?

Reply:
@abstractj _security-checks_ was rename to _scanners_ and default values are fine: _**vuln,secret**_

Sources => 
- https://github.com/aquasecurity/trivy/discussions/3519
- https://github.com/aquasecurity/trivy-action?tab=readme-ov-file#inputs